stratify/_vinterp.pyx

```diff
@@ -181,196 +155,262 @@ cdef long gridwise_interpolation(double[:] z_target, double[:] z_src,
                 break
 
         if extrapolating == 0 or sign_after == 0:
-            interpolation_kernel(i_src, z_src, fz_src, z_current,
+            interpolation.kernel(i_src, z_src, fz_src, z_current,
                                  fz_target[:, i_target])
         else:
-            extrapolation_kernel(extrapolating, z_src, fz_src, z_current,
```

Comment:
The ```extrapolating``` kwarg looks wrong to me. I think this is a bug that would have been very hard to test without this change.